octoprint_slic3r/profile.py

```diff
@@ -239,11 +261,18 @@ def to_slic3r_ini(cls, profile, path, display_name=None, description=None):
 	@classmethod
 	def convert_value(cls, key, value, default, sep=","):
 		try:
-			if key in float_or_percentage:
+			'''Some defaults can be instances of int whereas its values are instances of float. 
+			If that happens, change default to float so as to override default value without getting errors'''
+			if isinstance(default,int) and value.count('.')==1:
```

Comment:
Whitespace. You are still keeping your statements way too dense. Adding a space around operators and assignements really doesn't hurt and helps a lot with readability:

``` python
if isinstance(default, int) and value.count(".") == 1:
    default = float(default)
```